web/src/main/java/cz/kb/openbanking/adaa/example/springboot/web/model/StatementModel.java

```diff
@@ -0,0 +1,148 @@
+package cz.kb.openbanking.adaa.example.springboot.web.model;
+
+import java.time.LocalDate;
+import javax.annotation.Nullable;
+
+/**
+ * Represents account statement.
+ *
+ * @author <a href="mailto:aleh_kuchynski@kb.cz">Aleh Kuchynski</a>
+ * @since 1.1
+ */
+public class StatementModel {
```

Comment:
Can lombok (with @NonNull etc.) be used instead of getters, setters, null checks and throwing IllegalArgumentExceptions?

Reply:
The main idea of this repo is to show to our clients how to use ADAA API and ADAA SDK, so I'd like to make it self-describing as much as possible. Lombok is excellent tool, but hides a lot of code and descriptions under the hood. And I also tried to use a minimum of third-party libraries, because not all of developers should know them, however we want this project to be useful and understandable for all of clients